pages/api/comments/index.js

```diff
@@ -0,0 +1,24 @@
+import dbConnect from "../../../db/connect";
+import Comment from "../../../db/models/Comment";
+
+export default async function handler(request, response) {
+  await dbConnect();
+
+  if (request.method === "GET") {
+    const comments = await Comment.find();
+    return response.status(200).json(comments);
+  }
+
+  if (request.method === "POST") {
+    try {
+      const commentData = request.body;
+      const comment = new Comment(commentData);
+      await comment.save();
```

Comment:
In case you want to take the "DRY" approach, this also works just fine. You can use the create method, it creates and saves the new comment in one step.

`const commentData = request.body;
await Comment.create(commentData);`